pallets/stateful-storage/Cargo.toml

```diff
@@ -30,29 +30,14 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", default-features
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
 # Frequency related dependencies
 common-primitives = { default-features = false, path = "../../common/primitives" }
+twox-hash = { version = "1.6.3", default-features = false, features = ["digest_0_10"] }
 
 [dev-dependencies]
 env_logger = "0.10.0"
 pretty_assertions = "1"
 
 [features]
 default = ['std']
-runtime-benchmarks = [
-  "frame-benchmarking/runtime-benchmarks",
-  "frame-support/runtime-benchmarks",
-  "frame-system/runtime-benchmarks",
-  "common-primitives/runtime-benchmarks"
-]
-std = [
-  'codec/std',
-  'scale-info/std',
-  'sp-std/std',
-  'sp-core/std',
-  'sp-io/std',
-  'sp-runtime/std',
-  'frame-support/std',
-  'frame-system/std',
-  'frame-benchmarking/std',
-  'common-primitives/std'
-]
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks", "frame-support/runtime-benchmarks", "frame-system/runtime-benchmarks", "common-primitives/runtime-benchmarks"]
```

Comment:
another nit

Reply:
`cargo add` rewrites this as a single line, FYI.
Fixed.